elastic/kibana
# [Infra] Add infra custom dashboards feature flag to infra settings page

Closes #180386 
⚠️ To be merged after [#182859](https://github.com/elastic/kibana/issues/182859) is done
## Summary

This PR enables custom dashboards by default and adds the flag to infra settings:

<img width="1887" alt="image" src="https://github.com/elastic/kibana/assets/14139027/018aae99-a488-49dd-87ba-db0bcbc4caf6">


## Testing
- Go to infra settings page and check if the custom dashboards flag is enabled
- Go to the hosts view and select a host:  the dashboards tab should be visible
- Compare the flags with advanced settings 
- Disable the flag, go to the hosts view and select a host: the dashboards tab should not be visible
- Enable the flag, go to the hosts view and select a host: the dashboards tab should be visible


https://github.com/elastic/kibana/assets/14139027/f788c019-4fdb-416b-8fdb-b5cb39b8a2fc
